src/server.h

```diff
@@ -1148,7 +1153,9 @@ struct sharedObjectsStruct {
     *select[PROTO_SHARED_SELECT_CMDS],
     *integers[OBJ_SHARED_INTEGERS],
     *mbulkhdr[OBJ_SHARED_BULKHDR_LEN], /* "*<value>\r\n" */
-    *bulkhdr[OBJ_SHARED_BULKHDR_LEN];  /* "$<value>\r\n" */
+    *bulkhdr[OBJ_SHARED_BULKHDR_LEN],  /* "$<value>\r\n" */
+    *subscribelocalbulk,
+    *unsubscribelocalbulk;
```

Comment:
Nit pick, i'd rather declare these above `select`, so we don't have to modify the `bulkhdr` line.